hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/transactionhandler/CryptoDeleteAllowanceTransactionHandler.java

```diff
@@ -52,12 +62,18 @@ public void updateTransaction(Transaction transaction, RecordItem recordItem) {
             return;
         }
 
-        for (var nftAllowance : recordItem.getTransactionBody().getCryptoDeleteAllowance().getNftAllowancesList()) {
-            var tokenId = EntityId.of(nftAllowance.getTokenId());
-            for (var serialNumber : nftAllowance.getSerialNumbersList()) {
+        List<NftRemoveAllowance> nftAllowances = recordItem.getTransactionBody().getCryptoDeleteAllowance().getNftAllowancesList();
+        for (int i = 0; i < recordItem.getTransactionBody().getCryptoDeleteAllowance().getNftAllowancesCount(); i++) {
+            NftRemoveAllowance nftRemoveAllowance = nftAllowances.get(i);
```

Comment:
nit, can revert these lines. I think previously you made the change because `i` was used as the log index

```suggestion
        for (var nftAllowance : recordItem.getTransactionBody().getCryptoDeleteAllowance().getNftAllowancesList()) {
```

Reply:
You're right. Reverted.